### ISSUE TITLE
License

### ISSUE THREAD

**Issue by leouieda** [opengeophysics/2018-agu-oss#1]
We should have a license on this repo. I was thinking CC-BY for the text, etc, and BSD/MIT for the code. Any thoughts?

**Comment by krischer** [comment]
I agree with both - these are the most open choices. I don't have a preference regarding MIT/BSD - they are compatible in any case and the difference between both are marginal.

**Comment by lheagy** [comment]
I agree with both CC-BY and I am open to either MIT/BSD
